storybook/components/storybook-wrapper/src/StoryRowHeader.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from "react"
+import classNames from "classnames"
+import styles from "./styles.scss"
+
+export interface StoryRowHeaderProps {
+  children?: React.ReactNode
```

Comment:
```suggestion
  children: React.ReactNode
```

Then we shouldn't need `if (!children) return null` :)